kibana/templates/deployment.yaml

```diff
@@ -19,6 +19,18 @@ spec:
         app: kibana
         release: {{ .Release.Name | quote }}
     spec:
+      {{- if .Values.securityContext }}
+      securityContext: 
+        {{- if .Values.securityContext.runAsUser }}
```

Comment:
Instead of hardcoding two of the available options can you instead just do a `toYaml` so it is possible to set all of the [securityContext options](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#podsecuritycontext-v1-core)

Reply:
done